### PR TITLE
Add initial corpus files

### PR DESCRIPTION
**Pull request by mingxguo27**
This is a JSON file containing the initial seed inputs I used for fuzzing CTAP2 commands and helped a lot in the fuzzing process (coverage 400 -> 1500). The python script parses it into a corpus.

### CODE REVIEW

File: fuzz/ctap2_commands_parameters_corpus.json [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        "cbor": "map {\"a\": 1, \"b\": [2, 3]}",
+        "description": "cbor value"
+    },
+    {
+        "hex": "bf6346756ef563416d7421ff",
+        "cbor": "indefinite length map {\"Fun\": true, \"Amt\": -2}",
+        "description": "cbor value"
+    },
+    {
+        "hex": "",
```
Comment (gendx) [inline fuzz/ctap2_commands_parameters_corpus.json:93]: Why is the hex empty when the CBOR is not?

Comment (mingxguo27) [reply fuzz/ctap2_commands_parameters_corpus.json]: Ups my bad. Fixed.

File: fuzz/make_corpus.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import json
+import argparse
```
Comment (kaczmarczyck) [inline fuzz/make_corpus.py:2]: Please reorder. (Also, we probably need a Python linter if we start using Python.)

**Review by kaczmarczyck — CHANGES_REQUESTED** [review]

File: fuzz/make_corpus.py (hunk 2 above)
Comment (mingxguo27) [reply fuzz/make_corpus.py:2]: There is, but it didn't detect this somehow :(

**Review by kaczmarczyck — APPROVED** [review]

**Pull request merged**